packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -160,3 +142,23 @@
 .dataviews-view-grid__media--clickable {
 	cursor: pointer;
 }
+
+
+.dataviews-view-grid__row {
+	.dataviews-view-grid__row__gridcell {
+		outline: 0;
+		outline-style: solid;
+		// outline-offset: calc(-1 * var(--wp-admin-border-width-focus));
+		border-radius: $grid-unit-10;
+
+		&[data-focus-visible] {
+			outline-color: var(--wp-admin-theme-color);
+			outline-width: var(--wp-admin-border-width-focus);
+		}
+
+		&:hover {
+			outline-color: rgba($black, 0.3);
+			outline-width: var(--wp-admin-border-width-focus);
+		}
```

Comment:
The outer hover / focus states seem a bit awkward in terms of border-to-content spacing, and border radius:

![Screenshot 2024-12-30 at 20 02 28](https://github.com/user-attachments/assets/b4aef129-0d1f-443d-a618-034b72d2c41c)